testsuite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Use default azure-cli version 2.29.0 if no version is specified in build
+ARG AZURE_CLI_VERSION=2.29.0
+FROM mcr.microsoft.com/azure-cli:${AZURE_CLI_VERSION}
+
+RUN /usr/bin/curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl \
+    && chmod +x ./kubectl  \
+    &&  mv ./kubectl /usr/local/bin/kubectl
+
+# Install python3
+RUN apk add --update python3
+
+COPY kustomization.yaml /
+
+COPY launch.yaml /
```

Comment:
Why are we copying launch.yaml & kustomization.yaml inside the launcher container? They are not required

Reply:
done